Controller/ContentTypeController.php

```diff
@@ -127,6 +140,8 @@ public function viewContentTypeGroupAction($contentTypeGroupId)
             )->createView();
 
             $canDeleteById[$contentTypeId] = $canDelete && !$this->contentTypeService->isContentTypeUsed($contentType);
+
+            $languageById[$contentTypeId] = $this->getVerifiedContentTypeLanguageCode($contentType);
```

Comment:
Ok, I see I misunderstood the code a bit, id is referring to content type, so lets take prioritized languages into account, then we end up with something like

```php
function getPrioritizedLanguage($languageCodes, $fallbackLanguageCode)
{
    // the first possible natch in $this->prioritizedLanguages, fallback to $fallbackLanguageCode
}
```


use:
```php
 $languageById[$contentTypeId] = $this->getPrioritizedLanguage($languages = array_keys($contentType->names), $languages[0]);
```